### PR TITLE
Task 4 Hvisevich

### PR DESCRIPTION
**Pull request by Vitaly-Khvisevich**
This is my decision task №4

### CODE REVIEW

File: Tasks/Task4/Hvisevich [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def Summa(S, Summ=None):
+    Summ=Summ or 0
+    for i in range(len(S)):
+        if type(S[i])==list:
+            S[i]=Summa(S[i])
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Task4/Hvisevich:5]: Are you aware that you basically destroyed the list structure by such a assignments?

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]
Please implement another approach that will leave the initial list intact.

**Comment by Vitaly-Khvisevich** [comment]
changed the text in the code check please

**Review by MikitaTsiarentsyeu — APPROVED** [review]
it's good enough now, your score is 9.0

**Pull request merged**